src/ORM/BehaviorRegistry.php

```diff
@@ -67,6 +67,19 @@ public function __construct($table = null)
     }
 
     /**
+     * Get loaded behavior instance.
+     *
+     * @param string $name Name of behavior.
+     * @return \Cake\ORM\Behavior|object|null
+     */
+    public function __call($name, $params)
+    {
+        $name = substr($name, 3);
+
+        return $this->get($name);
```

Comment:
This seems like it would limit our options in the future. Shouldn't we check the prefix and throw an error if it is not get?

Reply:
Sure a possiblity.